c-dependencies/js-compute-runtime/builtins/transform-stream.cpp

```diff
@@ -222,7 +224,7 @@ bool ExtractStrategy(JSContext *cx, JS::HandleValue strategy, double default_hwm
     if (!JS::ToNumber(cx, val, hwm)) {
       return false;
     }
-    if (mozilla::IsNaN(*hwm) || *hwm < 0) {
+    if (std::isnan(*hwm) || *hwm < 0) {
```

Comment:
Between Mozilla Gecko 110 and 112, they removed mozilla::IsNaN - I've replaced the call with its `std` equivalent, which is also what is now used in Gecko's codebase